kubernetes-replicator.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: kubernetes-replicator
+  version: 2.10.2
+  epoch: 0
+  description: Kubernetes controller for synchronizing secrets & config maps across namespaces
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/mittwald/kubernetes-replicator
+      tag: v${{package.version}}
+      expected-commit: 8f4de965a57b974b446fae30048269894fd4ff3d
+
+  - uses: go/build
+    with:
+      packages: .
+      ldflags: |
+        -s -w
+        -X main.version=${{package.version}}
+      output: kubernetes-replicator
+
+  - uses: strip
```

Comment:
```suggestion

```
this can be removed as well given we already have `-w` via go/build pipeline.

Reply:
Thank you. Now resolved.